### ISSUE TITLE
select : Screen Reader is reading two times when user selects a value from combo-box

### ISSUE THREAD

**Issue by SMuraliWipro** [angular/components#26480]
### Is this a regression?

- [X] Yes, this behavior used to work in the previous version

### Description

This issue is seen in select component example. Screen reader is reading two times when user selects a values from combo-box.

Example link: [https://v11.material.angular.io/components/select/overview]

### Reproduction

Steps to reproduce:
1. Go to example link : [https://v11.material.angular.io/components/select/overview]
2. Open NVDA screen reader and reach combo-box by hitting 'tab' key.
3. Select a value from combo-box

### Expected Behavior

Selected value should be understandable to screen reader users. 

### Actual Behavior

Selected values reading multiple times by screen reader.

### Environment

- Angular:
- CDK/Material:
- Browser(s):
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by ArvindWip** [comment]
Hi Team, 
Please provide us the solution for the issue. It would be helpful to get an answer sooner. Thanks.

Arvind

**Comment by zarend** [comment]
Hello,

Thank you for reporting this issue. This looks like a user experience issue, but I wouldn’t expect it to block users from completing a workflow. For that reason, I’m updating the priority to P3 and wouldn’t consider this a GAR blocker. Please let us know if you have concerns and feel free “+1” if you are impacted.

Best,
Zach

**Comment by zarend** [comment]
Hello,

Hello,

Thank you for reporting this issue. I wouldn’t expect this issue to block users from completing a workflow. For that reason, I’m updating the priority to P3. Please let us know if you have further concerns concerns and feel free to react with “👍” if you are impacted.

Best,
Zach

**Comment by ArvindWip** [comment]
👍

**Comment by crisbeto** [comment]
We are closing this issue, because we've determined, using a semi-automated process, that it is no longer relevant, has low engagement, or it does not have enough information to proceed. Feel free to comment if you think the issue should be reopened, or open a new issue with the latest version of our code instead.
